src/cookbook/games/achievements-leaderboard.md

```diff
@@ -0,0 +1,352 @@
+    the game service.
+
+    ```dart
+    try {
+      await GamesServices.signIn();
+    } on PlatformException catch (e) {
+      // ... deal with failures ...
+    }
+    ```
+
+The sign in happens in the background. It takes several seconds, so
+don't call `signIn()` before `runApp()` or the players will be forced to
+stare at a blank screen every time they start your game.
+
+The API calls to the `games_services` API can fail for a multitude of
+reasons. Therefore, every call should be wrapped in a try-catch block as
+in the previous example. The rest of this recipe omits exception
+handling for clarity.
+
+{{site.alert.note}}
+  It's a good practice to create a controller. This would be a
+  `ChangeNotifier`, a bloc, or some other piece of logic that wraps around
+  the raw functionality of the `games_services` plugin.
+{{site.alert.end}}
+
+
+## 3. Unlock achievements
+
+1.  Register achievements in Google Play Console and App Store Connect,
+    and take note of their IDs. Now you can award any of those
+    achievements from your Dart code:
+
+    ```dart
+    await GamesServices.unlock(
+      achievement: Achievement(
+        androidID: 'your android id',
+        iOSID: 'your ios id',
+      ),
+    );
+    ```
+
+    The player's account on Google Play Games or Apple Game Center now
+    lists the achievement.
+
+2.  To display the achievements UI from your game, call the
+    `games_services` API:
+
+    ```dart
+    await GamesServices.showAchievements();
+    ```
+
+    This displays the platform achievements UI as an overlay on your game.
+
+3.  To display the achievements in your own UI, use
+    `GamesServices.loadAchievements()`.
+
+## 4. Submit scores
+
+When the player finishes a play-through, your game can submit the result
+of that play session into one or more leaderboards.
+
+For example, a platformer game like Super Mario can submit both the
+final score and the time taken to complete the level, to two separate
+leaderboards.
+
+1.  In the first step, you registered a leaderboard in Google Play
+    Console and App Store Connect, and took note of its ID. Using this
+    ID, you can submit new scores for the player:
+
+    ```dart
+    await GamesServices.submitScore(
+      score: Score(
+        iOSLeaderboardID: 'some_id_from_app_store',
+        androidLeaderboardID: 'sOmE_iD_fRoM_gPlAy',
+        value: 100,
+      ),
+    );
+    ```
+
+    You don't need to check whether the new score is the player's
+    highest. The platform game services handle that for you.
+
+2.  To display the leaderboard as an overlay over your game, make the
+    following call:
+
+    ```dart
+    await GamesServices.showLeaderboards(
+      iOSLeaderboardID: "some_id_from_app_store",
+      androidLeaderboardID: "sOmE_iD_fRoM_gPlAy",
+    );
+    ```
+
+3.  If you want to display the leaderboard scores in your own UI, you
+    can fetch them with `GamesServices.loadLeaderboardScores()`.
+
+## 5. Complete example
```

Comment:
I'm not sure "Complete example" makes sense as the title of this step, perhaps something such as `5. Next steps` or similar?

Reply:
Good point, fixed